CRM/Contribute/Form/Search.php

```diff
@@ -163,6 +153,18 @@ public function setDefaultValues() {
         'Completed'
       );
     }
+
+    // The membership or contribution id could be set on the form if viewing
+    // an embedded block on ParticipantView or MembershipView.
+    $memberShipId = CRM_Utils_Request::retrieve('memberId', 'Positive', $this);
+    if (isset($memberShipId)) {
```

Comment:
@seamuslee001 Want to change variable name to camecased as is the norm elsewhere?

Reply:
As this is a backport let's not change it here - but it would be nice to see a follow up PR against master that tidies up the variable name - @seamuslee001